### PR TITLE
finetune message and fix quickpick styling

### PR DESCRIPTION
**Pull request by sandy081**

### CODE REVIEW

File: src/vs/platform/quickinput/browser/quickInput.ts [hunk 1]
```diff
@@ -1749,6 +1751,7 @@ export class QuickInputController extends Disposable {
 		ui.description1.style.display = visibilities.description && (visibilities.inputBox || visibilities.checkAll) ? '' : 'none';
 		ui.description2.style.display = visibilities.description && !(visibilities.inputBox || visibilities.checkAll) ? '' : 'none';
 		ui.checkAll.style.display = visibilities.checkAll ? '' : 'none';
+		ui.inputContainer.style.display = visibilities.inputBox ? '' : 'none';
```
Comment (sandy081) [inline src/vs/platform/quickinput/browser/quickInput.ts:1754]: cc @TylerLeonhardt

Comment (TylerLeonhardt) [reply src/vs/platform/quickinput/browser/quickInput.ts:1754]: What does this fix?

Comment (sandy081) [reply src/vs/platform/quickinput/browser/quickInput.ts:1754]: image

**Review by TylerLeonhardt — APPROVED** [review]

**Comment by TylerLeonhardt** [comment]
Thanks!

**Pull request merged**